tempto-examples/README.md

```diff
@@ -5,10 +5,13 @@ smoketest to ensure that you've setup everything properly.
 
 ##Setup 
 
- * You need a cluster with hadoop accessible by name hadoop-master and a presto cluster accessible by name presto-master. In case you want to use different host names (or service ports) you need to change ```src/main/resources/tempto-configuration.yaml``` accordingly.For more details please refer to the [Configuration section](../README.md).
- * Ensure that WebHDFS, Hive and Presto are running.
+### Hadoop cluster
 
-Also you need PSQL instances.
+ You need a cluster with hadoop accessible by name hadoop-master and a presto cluster accessible by name presto-master. In case you want to use different host names (or service ports) you need to change ```src/main/resources/tempto-configuration.yaml``` accordingly.For more details please refer to the [Configuration section](../README.md).
```

Comment:
or use src/main/resources/tempto-configuration-local.yaml to override host names.